x-pack/plugins/security_solution/public/common/lib/triggers_actions_ui/register_alerts_table_configuration.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Storage } from '@kbn/kibana-utils-plugin/public';
+import { AlertsTableConfigurationRegistryContract } from '@kbn/triggers-actions-ui-plugin/public';
+
+import { APP_ID } from '../../../../common/constants';
+import { getTimelinesInStorageByIds } from '../../../timelines/containers/local_storage';
+import { TimelineId } from '../../../../common/types';
+import { columns } from '../../../detections/configurations/security_solution_detections';
+
+const registerAlertsTableConfiguration = (
+  registry: AlertsTableConfigurationRegistryContract,
+  storage: Storage
+) => {
+  const timelineStorage = getTimelinesInStorageByIds(storage, [TimelineId.detectionsPage]);
+  const alertColumns = timelineStorage[TimelineId.detectionsPage].columns ?? columns;
```

Comment:
I think [the error](https://github.com/elastic/kibana/pull/130881#issuecomment-1111397593) is because of this

Reply:
I pushed a commit to use optional chaining when accessing the attribute. that should fix it.